schemas/mlkem_test_schema.json

```diff
@@ -0,0 +1,183 @@
+                "format": "HexBytes",
+                "description": "The encapsulation key derived from the seed"
+              },
+              "c": {
+                "type": "string",
+                "format": "HexBytes",
+                "description": "An input ciphertext"
+              },
+              "K": {
+                "type": "string",
+                "format": "HexBytes",
+                "description": "The output shared secret"
+              },
+              "result": {
+                "type": "string",
+                "description": "Test result",
+                "enum": [
+                  "valid",
+                  "invalid",
+                  "acceptable"
+                ]
+              },
+              "flags": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "description": "A list of flags"
+              }
+            }
+          }
+        }
+      },
+      "required": [
+        "source"
+      ]
+    },
+    "MLKEMEncapsTestGroup": {
+      "type": "object",
+      "description": "A test for Encaps",
+      "properties": {
+        "type": {
+          "enum": [
+            "MLKEMEncapsTest"
+          ]
+        },
+        "source": {
+          "$ref": "common.json#/definitions/Source"
+        },
+        "tests": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "tcId": {
+                "type": "integer",
+                "description": "Identifier of the test case"
+              },
+              "comment": {
+                "type": "string",
+                "description": "A brief description of the test case"
+              },
+              "ek": {
+                "type": "string",
+                "format": "HexBytes",
+                "description": "The encapsulation key"
+              },
+              "m": {
+                "type": "string",
+                "format": "HexBytes",
+                "description": "The ML-KEM.Encaps_internal m input"
+              },
+              "c": {
+                "type": "string",
+                "format": "HexBytes",
+                "description": "The output ciphertext"
+              },
+              "K": {
+                "type": "string",
+                "format": "HexBytes",
+                "description": "The output shared secret"
+              },
+              "result": {
+                "type": "string",
+                "description": "Test result",
+                "enum": [
+                  "valid",
+                  "invalid",
+                  "acceptable"
+                ]
+              },
+              "flags": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "description": "A list of flags"
+              }
+            }
```

Comment:
Maybe we could add some `required` fields for the testcases? It looks like:

```json
            "required": [
              "tcId", "ek", "c", "result"
            ]
```

works without error. `m` and `k` don't appear in all tests, and `flags`/`comments` seem OK to omit.

Reply:
Handled in the diff attached to the other comment thread.